hacks/install_k9s

```diff
@@ -14,26 +14,53 @@ function show_help () {
 function install () {
   local version=$1
   local download_url
-  local arch=$(uname -m | sed 's/x86_64/amd64/;s/aarch64/arm64/')
-
-  if [ -z "$version" ]
-  then # fetch latest
-    local latest_release=$(curl -sL https://api.github.com/repos/derailed/k9s/releases/latest)
-    version=$(echo "${latest_release}" | jq -r '.tag_name')
-    echo "installing latest version ${version}"
-    download_url=$(echo "${latest_release}" | jq -r --arg filename "k9s_Linux_${arch}.tar.gz" '.assets[] | select(.name == $filename) | .browser_download_url')
-  else
-    if [[ ! $version == v* ]]
-    then
-      version="v${version}"
+  local yellow="\033[0;33m"
+  local nc="\033[0m"
+  local arch
+  local platform
+  local pkg
+  local dest
+  local yellow="\033[0;33m"
+  local nc="\033[0m"
```

Comment:
The `yellow` and `nc` vars appear twice.

Reply:
No longer like that.